src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -825,6 +838,16 @@ subroutine InitHistory(this, bounds, carbon_type)
                avgflag='A', long_name='grain C to food', &
                ptr_patch=this%grainc_to_food_patch)
 
+		  this%leafc_to_biofuelc_patch(begp:endp) = spval
+          call hist_addfld1d (fname='LEAFC_TO_BIOFUELC', units='gC/m^2/s', &
+               avgflag='A', long_name='leaf C to biofuel C', &
+               ptr_patch=this%leafc_to_biofuelc_patch)
+
+		  this%livestemc_to_biofuelc_patch(begp:endp) = spval
```

Comment:
Please fix indentation of this line (see comment elsewhere about tabs vs. spaces).